phpcs.xml.dist

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<ruleset name="behat-phpserver">
+  <description>The coding standard for behat-phpserver.</description>
+
+  <file>src</file>
+  <file>tests</file>
+
+  <exclude-pattern>*/vendor/*</exclude-pattern>
+
+  <rule ref="PHPCS" />
```

Comment:
this should be using `https://github.com/djoos/Symfony2-coding-standard` and there is no need for this custom standards file.

Please install this standard here and remove this file. And please update all code to follow this new standard.